src/me/john000708/barrels/Barrels.java

```diff
@@ -232,27 +273,35 @@ public boolean onRightClick(ItemUseEvent itemUseEvent, Player player, ItemStack
             public boolean onRightClick(ItemUseEvent itemUseEvent, Player player, ItemStack itemStack) {
                 if (!SlimefunManager.isItemSimiliar(itemStack, ID_CARD, false)) return false;
                 Block clickedBlock = itemUseEvent.getClickedBlock();
-                ItemStack idCard = itemStack;
-                ItemMeta meta = idCard.getItemMeta();
-                List<String> lore = idCard.getItemMeta().getLore();
+                // No need to reference itemStack again in a new variable.
+                //ItemStack idCard = itemStack;
+                ItemMeta meta = itemStack.getItemMeta();
+                List<String> lore = itemStack.getItemMeta().getLore();
 
                 if (lore.get(0).equals("")) {
                     lore.set(0, ChatColor.translateAlternateColorCodes('&', "&0" + player.getUniqueId().toString()));
                     lore.set(1, ChatColor.translateAlternateColorCodes('&', "&fBound to: " + player.getName()));
                     meta.setLore(lore);
-                    idCard.setItemMeta(meta);
+                    itemStack.setItemMeta(meta);
                     player.sendMessage(ChatColor.GREEN + "ID Card bound.");
+                    return true;
```

Comment:
This return statement is redundant, since you return true in the end anyway.

Reply:
But it's return false and we surely want to return true after those happen.

Reply:
See Line 304...
You added another return true; statement.

Reply:
This btw is still redundant.